src/components/Card/Card.js

```diff
@@ -12,39 +13,55 @@ Card.propTypes = {
 }
 
 export default function Card({
+  id,
   text,
   title,
   image,
   address,
   openingHours,
   entry,
+  handleBookmark,
+  isFavorite,
 }) {
   const [isExpanded, setIsExpanded] = useState(false)
 
   return (
-    <WrapperCardList>
-      <img src={image} alt="" />
-      <h2>{title}</h2>
+    <>
+      <CardWrapper>
+        <Bookmarks
+          isBookmarked={isFavorite}
+          aria-label=""
+          onClick={() => handleBookmark(id)}
+        />
+        <CardList>
+          <img src={image} alt="" />
+          <h2>{title}</h2>
 
-      {isExpanded && (
-        <div>
-          <p>{text}</p>
-          <p>{address}</p>
-          <p>{entry}</p>
-          {openingHours.map(openingHours => (
-            <p>{openingHours}</p>
-          ))}
-        </div>
-      )}
+          {isExpanded && (
+            <div>
+              <p>{text}</p>
+              <p>{address}</p>
+              <p>{entry}</p>
+              {openingHours.map(openingHours => (
+                <p>{openingHours}</p>
+              ))}
+            </div>
+          )}
 
-      <ButtonExpansion onClick={() => setIsExpanded(!isExpanded)}>
-        {isExpanded ? 'Hide details' : 'Show details'}
-      </ButtonExpansion>
-    </WrapperCardList>
+          <ButtonExpansion onClick={() => setIsExpanded(!isExpanded)}>
+            {isExpanded ? 'Hide details' : 'Show details'}
+          </ButtonExpansion>
```

Comment:
This name lets me think that it is an expansion of a button but I think it actually expands other Elements
```suggestion
          <ExpandButton onClick={() => setIsExpanded(!isExpanded)}>
            {isExpanded ? 'Hide details' : 'Show details'}
          </ExpandButton>
```